src/types/entities/guild_member.rs

```diff
@@ -9,17 +9,20 @@ use crate::types::Shared;
 use crate::types::{entities::PublicUser, Snowflake};
 
 #[derive(Debug, Deserialize, Default, Serialize, Clone)]
+#[cfg_attr(feature = "sqlx", derive(sqlx::FromRow))]
 /// Represents a participating user in a guild.
 ///
 /// # Reference
 /// See <https://discord-userdoccers.vercel.app/resources/guild#guild-member-object>
 pub struct GuildMember {
+    #[cfg_attr(feature = "sqlx", sqlx(skip))]
     pub user: Option<Shared<PublicUser>>,
     pub nick: Option<String>,
     pub avatar: Option<String>,
+    #[cfg_attr(feature = "sqlx", sqlx(skip))]
     pub roles: Vec<Snowflake>,
     pub joined_at: DateTime<Utc>,
-    pub premium_since: Option<DateTime<Utc>>,
+    pub premium_since: Option<String>,
```

Comment:
Probably also an oversight when merging?
```suggestion
    pub premium_since: Option<DateTime<Utc>>,
```